### PR TITLE
Wiki crud

### PR DESCRIPTION
**Pull request by alexromo3rd**
Hey Ben,

Just completed this checkpoint. Everything seems to work just fine. Let me know what you think. Thanks.

### CODE REVIEW

File: app/views/wikis/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+				<%= f.label :title %>
+				<%= f.text_field :title, class: 'form-control', placeholder: "Enter wiki title" %>
+			</div>
+			<div class="form-group">
+				<%= f.label :body %>
+				<%= f.text_area :body, rows: 8, class: 'form-control', placeholder: "Enter wiki body" %>
+			</div>
+			<%= f.submit "Save", class: 'btn btn-success' %>
+		<% end %>
+	</div>
```
Comment (bmneely) [inline app/views/wikis/edit.html.erb:16]: Got some funky tabs to spaces going on here. :)

File: spec/controllers/wikis_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET #destroy" do
+    it "returns http success" do
+      get :destroy
+      expect(response).to have_http_status(:success)
+    end
+  end
```
Comment (bmneely) [inline spec/controllers/wikis_controller_spec.rb:52]: Specs! Beautiful. :)

**Review by bmneely — APPROVED** [review]

File: app/views/wikis/edit.html.erb (hunk 1 above)
Comment (alexromo3rd) [reply app/views/wikis/edit.html.erb:16]: I need to look into why it does that. I noticed that when I edit a file using a text editor it always ends up looking like this when I upload it. However, if I use vim to edit a file the spaces come out just right when uploaded.

Comment (bmneely) [reply app/views/wikis/edit.html.erb:16]: That's cause vim is the best :D

**Pull request merged**